device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/IotHubTransport.java

```diff
@@ -101,6 +101,10 @@
     private final ProxySettings proxySettings;
     private SSLContext sslContext;
 
+    // Used to store the CorrelationCallbackMessage for a correlationId
+    private final static Map<String, CorrelatingMessageCallback> correlationCallbacks = new ConcurrentHashMap<>();
+    private final static Map<String, Object> correlationContext = new ConcurrentHashMap<>();
```

Comment:
How about "correlationCallbackContexts" to follow the naming patterns within this class?